tests/zfs-tests/tests/functional/fault/fault.cfg

```diff
@@ -51,3 +52,9 @@ if is_linux; then
 else
 	DEV_DSKDIR="/dev"
 fi
+
+export VDEV_FILES="/var/tmp/file-1 /var/tmp/file-2 /var/tmp/file-3 /var/tmp/file-4 /var/tmp/spare-1"
```

Comment:
Please use `$TEST_BASE_DIR"` for `/var/tmp` everywhere.  This way we can easily run the ZTS in different directories which may have more space.  This is important for the buildbot automated testing which sets this to make sure the tests use the ec2 "instance storage" instead of ebs volumes.

Reply:
Fixed